www.foia.gov/_sass/modules/_typeahead.scss

```diff
@@ -0,0 +1,40 @@
+.twitter-typeahead {
+  width: 100%;
+
+  .tt-menu {
+    background-color: #fff;
+    border: 0.0625rem solid #ccc;
+    border: 0.0625rem solid rgba(0, 0, 0, 0.2);
+    border-radius: 0.5rem;
+    box-shadow: 0 0.3rem 0.625rem rgba(0,0,0,.2);
+    cursor: pointer;
+    margin-top: 0;
+    max-height: 150px;
+    overflow-y: auto;
+    padding: 1rem 0;
+    width: 100%;
+  }
+
+  .tt-hint,
+  .tt-input {
+    margin: 0;
+  }
+
+  .tt-suggestion {
+    border-bottom: 0.0625rem solid #eee;
+    font-size: $h4-font-size;
+    line-height: 1.5rem;
+    padding: 0.5rem 1.2rem;
```

Comment:
Let's do `$space-1x $space-2x` for now to keep the sizes consistent with the rest of the spacing.